### DISCUSSION TITLE
KIE on wildreceipt dataset

### DISCUSSION THREAD

**Discussion by riyajatar93** [PaddlePaddle/PaddleOCR#12808] [Q&A]
I am trying to use KIE on wildreceipt dataset.
from this link
https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/doc/doc_en/algorithm_kie_sdmgr_en.md

but why i need a ground truth label during inference? 
shouldn't we pass the output of text detection and text recognition to KIE model as input ?

python3 tools/infer_kie.py -c configs/kie/kie_unet_sdmgr.yml -o Global.checkpoints=kie_vgg16/best_accuracy  Global.infer_img=./train_data/wildreceipt/1.txt

I will appreciate your help , i am stuck here

**Comment by jchain** [comment]
`./train_data/wildreceipt/1.txt` is not ground truth label. It contains the input image path and the output of text detection and recognition.
